cfpb/cfgov-refresh
# Make Bureau Structure download field optional

## Changes

- Sets `required=False` on the `BureauStructure` organism's `download_image` field
- As conditional to template to output the download button only if the field has a document selected

## Testing

1. Pull branch
1. Migrate
1. Edit the page to clear the field: http://localhost:8000/admin/pages/4611/edit/
1. Publish or preview and see that the button area goes away

## Checklist
